docs/pages/reference/account-kit/react/functions/createConfig.mdx

```diff
@@ -31,9 +31,12 @@ addPasskeyOnSignup: true,
 }
 
 const config = createConfig({
-transport: alchemy({ apiKey: "your_api_key" })
-chain: sepolia,
-ssr: true,
+  transport: alchemy({ apiKey: "your_api_key" })
+  chain: sepolia,
+  ssr: true,
+  sessionConfig: {
+    duration: 60 * 60, // 1 hour in seconds
```

Comment:
❗ this should be `expirationTimeMs` not `duration`

Reply:
also this file is generated and shouldn't be edited directly. this should go in the jsdoc comment in `createConfig.ts` to end up here.